docs/getting_started/how_to_update_mautic.rst

```diff
@@ -22,6 +22,39 @@ Before you commence updating Mautic, **please ensure that you have a tested back
 
 This means that you have downloaded the files and database of your Mautic instance, and you have re-created it in a test environment somewhere and tested that everything is working as expected. This is your only recourse if there are any problems with the update. Never update without having a working, up-to-date backup.
 
+Updating Mautic Core (with Composer based)
```

Comment:
```suggestion
Updating Mautic (Composer based installs)
```
I think this might be a better heading - see what you think! If you choose that, you'll need to update the underline to match the length.